hub/core/dataset/dataset.py

```diff
@@ -1589,12 +1590,15 @@ def tensorflow(
         Args:
             tensors (List, Optional): Optionally provide a list of tensor names in the ordering that your training script expects. For example, if you have a dataset that has "image" and "label" tensors, if ``tensors=["image", "label"]``, your training script should expect each batch will be provided as a tuple of (image, label).
             tobytes (bool): If ``True``, samples will not be decompressed and their raw bytes will be returned instead of numpy arrays. Can also be a list of tensors, in which case those tensors alone will not be decompressed.
+            fetch_chunks: See fetch_chunks argument in hub.core.tensor.Tensor.numpy()
```

Comment:
I don't think this decision should be exposed to the user, we should always fetch chunks in the context of tensorflow. What do you think @AbhinavTuli ?

Reply:
I run the benchmark again with larger images (coco) and there was still a significant speed-up (29 fps instead of 7 fps). I do however assume that getting the first sample might be really slow if the images are large if we always use chunks, so perhaps there should be an option to disable it at least.

Coco benchmark here: https://gist.github.com/daniel-falk/e827be6abf4eb81c7083ec1732b5bb39

Reply:
@daniel-falk With tensorflow, we know we are going to be fetching the whole dataset (unless there are step slices) so the time to download the first sample is not relevant imo.

Reply:
@farizrahman4u There are cases when I just want to take one or a few samples from my tensorflow dataset. One example is for visualization of a sample to debug augmentations, another example is when i need a "representable dataset" for integer quantization. In both cases I need to use the tensorflow dataset instead of the hub dataset since I want the transformations. But I agree with your point, it is no big issue if there are a few seconds of extra delay. For a 640x480x3 image, taking the first sample goes from 1 second to 5 second if using `fetch_chunks=True`.

Reply:
@daniel-falk interesting, in that case can you set the default for fetch chunks to True here?

Reply:
@farizrahman4u Yes, I think that is a good option.